apps/tangle-dapp/constants/polkadot.ts

```diff
@@ -1,85 +1,86 @@
 import { ApiPromise, ApiRx, WsProvider } from '@polkadot/api';
-import { u128 } from '@polkadot/types';
-import { formatBalance } from '@polkadot/util';
+import { BN, formatBalance } from '@polkadot/util';
 import { TANGLE_RPC_ENDPOINT as TESTNET_RPC_ENDPOINT } from '@webb-tools/webb-ui-components/constants';
 import { firstValueFrom } from 'rxjs';
 
-const TOKEN_UNIT = 'tTNT';
+export const TOKEN_UNIT = 'tTNT';
```

Comment:
This should be more explicit no? `TANGLE_TESTNET_TOKEN_UNIT`. Also why is this file named `polkadot`? Can we come up with a better name here?

Reply:
Since it mostly contains Polkadot API-related utility functions, I think the name is somewhat appropriate. I'll change it to `polkadotApiUtils` to make it more clear, although one thing we could improve is that it is placed under `constants/` but it is mainly based of utility functions.